cypress/e2e/ModelizerPage/SwitchView/delete.feature

```diff
@@ -0,0 +1,228 @@
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"inactive-tab\"]" is "default.plugin-test.js"
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"active-tab\"]" is "leto-modelizer.config.json"
+    And  I wait 1 second
+
+    When I click on "[data-cy=\"file-tabs-container\"] [data-cy=\"file-label-default.plugin-test.js\"]"
+    Then I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"active-tab\"]" is "default.plugin-test.js"
+    And  I expect active file content to contain "\"type\": \"image\""
+    And  I expect active file content to contain "\"type\": \"paper\""
+    And  I expect active file content to contain "\"name\": \"image-attachments\""
+    And  I expect active file content to contain "\"type\": \"Array\""
+
+    When I click on "[data-cy=\"modelizer-switch\"] [aria-pressed=\"false\"]"
+    Then I expect "[data-cy=\"modelizer-switch\"] [aria-pressed=\"true\"] [class=\"block\"]" is "Model"
+    And  I expect "[data-cy=\"modelizer-model-view-draw-root\"]" exists
+    And  I expect ".link" appear 1 time on screen
+
+    #  NOTE: must force click because the following two elements are covered by an overflowing svg element
+    When I force click on ".link"
+    And  I force click on "#remove-link"
+    Then I expect ".link" not exists
+
+    When I click on "[data-cy=\"modelizer-switch\"] [aria-pressed=\"false\"]"
+    Then I expect "[data-cy=\"modelizer-switch\"] [aria-pressed=\"true\"] [class=\"block\"]" is "Text"
+    And  I expect "[data-cy=\"modelizer-text-view\"]" exists
+    And  I expect active file content to contain "\"type\": \"image\""
+    And  I expect active file content to contain "\"type\": \"paper\""
+    But  I expect active file content to not contain "\"name\": \"image-attachments\""
+    And  I expect active file content to not contain "\"type\": \"Array\""
+
+  Scenario: Remove link attribute inside plugin file content (Text view) should remove the link between the two components (Model view)
+    Given I click on "[data-cy=\"component-definition-paper\"]"
+    And  I click on "[data-cy=\"component-definition-image\"]"
+
+    When I click on "[id^=\"paper_\"]"
+    And  I click on "[id=\"create-link\"]"
+    And  I click on "[id^=\"image_\"]"
+    Then I expect ".link" exists
+    
+    When I click on "[data-cy=\"modelizer-switch\"] [aria-pressed=\"false\"]"
+    Then I expect "[data-cy=\"modelizer-switch\"] [aria-pressed=\"true\"] [class=\"block\"]" is "Text"
+    And  I expect "[data-cy=\"modelizer-text-view\"]" exists
+    And  I expect "[data-cy=\"file-label-default.plugin-test.js\"]" appear 2 times on screen
+    And  I expect "[data-cy=\"file-label-leto-modelizer.config.json\"]" appear 2 times on screen
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"inactive-tab\"]" is "default.plugin-test.js"
+    And  I expect "[data-cy=\"file-tabs-container\"] [data-cy=\"active-tab\"]" is "leto-modelizer.config.json"
+    And  I wait 1 second
```

Comment:
and -> when
put it in the next paragraph

Reply:
done